BottomSheetDemo/Sources/User Interface/Screens/Root/RootViewController.swift

```diff
@@ -53,7 +53,11 @@ final class RootViewController: UIViewController {
         let viewController = ResizeViewController(initialHeight: 300)
         presentBottomSheetInsideNavigationController(
             viewController: viewController,
-            configuration: .default
+            configuration: BottomSheetConfiguration(
+                cornerRadius: 10,
+                pullBarConfiguration: .visible(.init(height: 50)),
```

Comment:
I'd expect `.default` appearance in Demo application, so can you revert this change?